util/release-odemis

```diff
@@ -97,15 +110,15 @@ if [[ "$BPVERSION" != "$PVERSION" ]]; then
     exit 1
```

Comment:
There is a TODO on line 96 (github just won't let me comment there): `# TODO: for a minor release, this is sufficient:`
That should now be easy to implement, although I'm not sure if it actually matters. Could you either implement the comment, or remove the TODO?

Reply:
Actually, I think this is more like "the first time you do a release of this branch on this computer". As it's not hurting to always explicitly do "--set-upstream upstream" everytime, and it might not always be the same person/computer that does the release, I've dropped the TODO, and adjusted the comment.